edx_api/grades/init_test.py

```diff
@@ -0,0 +1,104 @@
+"""
+Test handling of responses from grades api.
+"""
+
+import json
+import os
+from unittest import TestCase
+
+import requests_mock
+import six
+
+from edx_api import enrollments, grades
+from edx_api.client import EdxApi
+
+
+class GradesApiTestCase(TestCase):
+    """
+    Test handling of mocked API responses in Grades API client.
+    """
+
+    base_url = "https://edx.example.com"
+
+    def setUp(self):
+        super(GradesApiTestCase, self).setUp()
+
+        with open(
+            os.path.join(
+                os.path.dirname(__file__),
+                "../enrollments/fixtures/user_enrollments.json",
+            )
+        ) as file:  # pylint: disable=redefined-builtin
```

Comment:
The file builtin is long-deprecated, and removed in python 3, so it's perfectly reasonable to redefine it.